returnn/torch/engine.py

```diff
@@ -466,6 +477,10 @@ def train_epoch(self):
                 accumulated_losses_dict += losses_dict
                 accumulated_inv_norm_factors_dict += inv_norm_factors_dict
                 eval_info = self._maybe_extend_losses_info(losses_dict / inv_norm_factors_dict)
+                if self.config.bool("log_stepwise_padding_ratio", False):
```

Comment:
Do you really want to use this? I will not use this. So then I would just remove this.

Reply:
Yes, I am going to use this for the bucketing experiments.

Reply:
Yeah I can also keep the changes local to my RETURNN copy.

Reply:
You can also do this just inside your `train_step` function.